amlip_cpp/test/manual/dds/rpcclient.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file rpcclient.cpp
+ *
+ */
+
+#include <thread>
+
+#include <cpp_utils/Log.hpp>
+
+#include <amlip_cpp/types/id/AmlipIdDataType.hpp>
+#include <amlip_cpp/types/model/ModelDataType.hpp>
+#include <amlip_cpp/types/model/ModelSolutionDataType.hpp>
+#include <dds/Participant.hpp>
+
+#include <dds/network_utils/model_manager.hpp>
+
+int main(
+        int argc,
+        char** argv)
+{
+    // Activate log
+    eprosima::utils::Log::SetVerbosity(eprosima::utils::Log::Kind::Info);
+
+    logUser(AMLIPCPP_MANUAL_TEST, "Starting Manual Test RPC Client execution. Creating Participant...");
+
+    {
+        // Create Participant
+        eprosima::amlip::types::AmlipIdDataType id({"RPC-CLIENT"}, {66, 11, 77, 44});
+        eprosima::amlip::dds::Participant participant(id);
+
+        logUser(AMLIPCPP_MANUAL_TEST, "Created Participant: " << participant << ". Creating Client...");
+
+        // Create RPC Client
+        std::shared_ptr<
+            eprosima::amlip::dds::RPCClient<eprosima::amlip::types::AmlipIdDataType,
+            eprosima::amlip::types::AmlipIdDataType>> client =
+                participant.create_rpc_client<eprosima::amlip::types::AmlipIdDataType,
+                        eprosima::amlip::types::AmlipIdDataType>("manual_test_topic");
+
+        // Data to request
+        eprosima::amlip::types::AmlipIdDataType data("MobileNet V1");
```

Comment:
Super NIT (I know it's just a test): I would keep everything in the DDS layer agnostic, i.e. I would not mention models.